include/ibamr/IBFEMethod.h

```diff
@@ -1162,47 +866,45 @@ class IBFEMethod : public IBStrategy
     /// Minimum ghost cell width.
     SAMRAI::hier::IntVector<NDIM> d_ghosts = 0;
 
-    /// Vectors of pointers to the systems for each part (for position, velocity, force
-    /// density, sources, and body stress normalization).
-    std::vector<libMesh::ExplicitSystem*> d_X_systems, d_U_systems, d_F_systems, d_Q_systems, d_Phi_systems;
+    /// Vectors of pointers to the systems for each part (for sources; other
+    /// systems are handled by the base class).
+    std::vector<libMesh::ExplicitSystem*> d_Q_systems;
 
     /*!
-     * Object managing access to libMesh system vectors for the structure position.
+     * Object managing access to libMesh system vectors for the source/sink
+     * strength.
      */
-    std::unique_ptr<IBTK::LibMeshSystemIBVectors> d_X_vecs;
+    std::unique_ptr<IBTK::LibMeshSystemVectors> d_Q_vecs;
 
     /*!
-     * Object managing access to libMesh system vectors for the structure velocity.
+     * Convenience pointer for object managing access to libMesh system vectors
+     * for the position.
      */
-    std::unique_ptr<IBTK::LibMeshSystemIBVectors> d_U_vecs;
+    IBTK::LibMeshSystemIBVectors* d_X_IB_vecs;
 
     /*!
-     * Object managing access to libMesh system vectors for the structure force.
+     * Convenience pointer for object managing access to libMesh system vectors
+     * for the velocity.
      */
-    std::unique_ptr<IBTK::LibMeshSystemIBVectors> d_F_vecs;
+    IBTK::LibMeshSystemIBVectors* d_U_IB_vecs;
 
     /*!
-     * Vectors of pointers to the fluid source or sink density vectors. All of
-     * these vectors are owned by the libMesh::System objects except for the
-     * ones in d_Q_IB_ghost_vecs, which are owned by the FEDataManager
-     * objects.
+     * Convenience pointer for object managing access to libMesh system vectors
+     * for the force.
      */
-    std::vector<libMesh::PetscVector<double>*> d_Q_half_vecs, d_Q_rhs_vecs, d_Q_IB_ghost_vecs;
```

Comment:
I deliberately left all the fluid source/sink vectors alone since we don't have any tests - are you sure that we should proceed with this change at this time? I would prefer to handle this later.

Reply:
Yes let's do it now.